packages/x-charts-pro/src/internals/plugins/useChartProExport/useChartProExport.ts

```diff
@@ -0,0 +1,52 @@
+import { ChartPlugin } from '@mui/x-charts/internals';
+import { printChart } from './print';
+import { ChartPrintExportOptions, UseChartProExportSignature } from './useChartProExport.types';
+
+function waitForAnimationFrame() {
+  let resolve: (_: void) => void;
+
+  const promise = new Promise((res) => {
+    resolve = res;
+  });
+
+  window.requestAnimationFrame(() => {
+    resolve();
+  });
+
+  return promise;
+}
+
+export const useChartProExport: ChartPlugin<UseChartProExportSignature> = ({
+  chartRootRef,
+  instance,
+}) => {
+  const print = async (options?: ChartPrintExportOptions) => {
+    const chartRoot = chartRootRef.current;
+
+    if (chartRoot) {
+      const enableAnimation = instance.disableAnimation();
```

Comment:
Not sure to get why you need the "no print" media query if you have this disable animation